book/Zapp-Pipes/12.-Datasource-for-rokuTV.md

```diff
@@ -0,0 +1,83 @@
+# Data Source Provided (DSP) for Roku-TV
+
+**This documentation is in beta**
+
+The Roku-TV system cannot use the current DSP solution, because the javascript code can't be parsed by Roku. This documentation explain how to solve this problem in a few steps.
```

Comment:
It is not amazing to start a document describing what something cannot do.

javascript -> JavaScript

explain -> explains

Reply:
@aethiss let's talk about this one first.